src/main/java/pl/edu/pw/ee/catering/model/order/service/impl/OrderImpl.java

```diff
@@ -32,4 +33,9 @@ public void changeOrderStatus(Long id, OrderStatus status) {
     private void changeOrderStatus(OrderWithDetails order, OrderStatus status) {
         order.setStatus(status);
     }
+    
+    @Override
+    public OrderList getOrderList(Long id) {
+        return new OrderList(orderRepository.findByStatusIsNotFinished(id));
```

Comment:
To jest 'kontrowersyjne' - ale moim zdaniem zgodne ze specyfikacją.  
Dokument jasno mówi: 
>>lista zamówień
zbiór wszystkich zamówień, które są niezrealizowane, bądź są w trakcie realizacji (nie
osiągnęły statusu końcowego).  
  
Oraz:  
>> status końcowy zamówienia
status zamówienia “zakończone” wyzwalane przez klienta po odebraniu zamówienia.  
  
 Czyli chcemy wyświetlać wszystkie zamówienia, które nie mają statusu zakończone.